spec/spec.md

```diff
@@ -836,7 +877,298 @@ processing-related rules above:
+          the same as the [[Ref:Subject]] of the attributes identified by
+          the other values in the `field_id` array.
+
+
+  The `same_subject` property would be used by a [[ref:Verifier]] to require
+  that certain provided inputs be about the same [[Ref:Subject]]. For
+  example, a [[ref:Presentation Definition]] might contain an
+  [[ref:Input Descriptor]] which calls for a street address from a driver
+  license [[ref:Claim]] and another [[ref:Input Descriptor]] which calls
+  for a name from a birth certificate [[ref:Claim]]. Using the
+  `same_subject` property, [[ref:Verifier]] would be able to require that
+  the [[Ref:Subject]] of the street address attribute [[ref:Claim]] is the
+  same as the [[Ref:Subject]] of the name attribute.
+
+
+## Credential Status Constraint Feature
+
+- The _constraints object_ ****MAY**** contain a `statuses` property. If
+present, its value ****MUST**** be an object that includes one or more of
+the following status properties:
+   - `active` - A credential is active if it is not revoked, expired,
+     suspended, or in any type of deactivated state.
+   - `suspended` - A credential is suspended if the Issuer has published an
+     explicit signal that the credential is in an inactive state and
+     ****should not**** currently be relied upon, but may become active
+     again in the future.
+   - `revoked` - A credential is revoked if the Issuer has published an
+     explicit signal that the credential in question ****should not**** be
+     relied upon going forward as an accurate reflection of the Issuer's
+     statements about the [[Ref:Subject]] within the scope of the
+     credential.
+
+The values of all status properties are objects, composed as follows:
+  - _status objects_ ****MUST**** include a `directive` property, and its
+    value ****MUST**** be one of the following strings:
+      - `required` - the credential ****MUST**** be of the specified
+        status.
+      - `allowed` - the credential ****MAY**** be of the specified status.
+      - `disallowed` - the credential ****MUST NOT**** be of the specified
+        status.
+    ```json
+      "statuses": {
+        "active": {
+          "directive": "required"  // other values: "allowed", "disallowed"
+        },
+        "suspended": {...},
+        "revoked": {...}
+      }
+    ```
+  ::: note
+  There is no assumed direct mapping between these values and a
+  corresponding status object in the underlying credentials. On the
+  contrary, the encoding and decoding of a credential status (which may
+  include fetching remote status information or cryptographic operations) is
+  an implementation detail which takes place at a lower layer of abstraction
+  and in accordance with the supported verifiable credential formats and
+  presentation protocols.
+  :::
+
+
+## JSON-LD Framing Feature
+
+The JSON-LD Framing [[ref:Feature]] extends [ref:Presentation Definition]] to support
+extended with the JSON-LD document framing.
```

Comment:
Does it need mentioning that this depends on `limit_disclosure` being preferred or required?

Reply:
It might help the LD-unfamiliar reader to add a sentence like: 
`LD framing objects are required for some forms of selective disclosure presentation.`
I'm not sure it is precise to describe it as a dependence in either direction, though-- you might want to include a framing object regardless of selective disclosure, just because you like RDF. who knows why people do the things they do!

Reply:
Yeah agree. Although you mention `some forms of selective disclosure` already, maybe best to replace `required' with `are being`, to make it more clear that frames typically are optional even in case of selective disclosure in PEX